catalog/dags/providers/provider_workflows.py

```diff
@@ -192,6 +193,10 @@ def __post_init__(self):
 
 
 PROVIDER_WORKFLOWS = [
+    ProviderWorkflow(
+        start_date=datetime(2023, 10, 1),
+        ingester_class=AucklandMuseumDataIngester,
```

Comment:
```suggestion
        ingester_class=AucklandMuseumDataIngester,
        schedule_string="@daily",
        dated=True,
```

We just need to add these in to make the DAG dated 🎉